README.md

```diff
@@ -1643,7 +1644,30 @@ Create a new function that calls _func_ with _thisArg_ and _args_.
   7.0 ✔  |  ✔ | 4.0 ✔ |  9.0 ✔ |  11.6 ✔ |  5.1 ✔  |
 
  **[⬆ back to top](#quick-links)**
+ 
+ ### _.debounce
+Create a new function that calls _func_ with _thisArg_ and _args_.
+
+  ```js
+ const debounce = (callback, time = 250, interval) => 
```

Comment:
You should use function instead arrow function otherwise this won't work in IE

Reply:
sure

Reply:
@ODudek  done